### ISSUE TITLE
Add model comparison metrics to interface

### ISSUE THREAD

**Issue by BartekPog** [BartekPog/AutoML-Library#5]
Currently, all classification models are compared by their prediction *accuracy*. It is acceptable in some cases, but in some imbalanced datasets it may lead to poor grid search decisions and not choosing the best model. 

Give user a chance to override the default *accuracy* metrics by others - for example, those already supported by `Grid Search` in `sklearn` library.

**Comment by BartekPog** [comment]
We can take advantage of the `GridSearchCV` scoring system. It may be either a **text** - eg. *accuracy* or a **callable** (scoring function) implemented by the user. To do that we just have to call `sklearn.metrics.make_scorer(custom_scoring_function)`. 

Here we can find more examples of [scoring in sklearn](https://scikit-learn.org/stable/modules/model_evaluation.html) and [make_scorer](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.make_scorer.html#sklearn.metrics.make_scorer).
